platforms/ruby/Dockerfile.t

```diff
@@ -25,7 +25,6 @@ RUN mkdir -p /app/heroku/node
 RUN curl -s https://s3pository.heroku.com/node/v$NODE_ENGINE/node-v$NODE_ENGINE-linux-x64.tar.gz | tar --strip-components=1 -xz -C /app/heroku/node
 ENV PATH /app/heroku/node/bin:$PATH
 
-ONBUILD WORKDIR /app/src
```

Comment:
It's already [here](https://github.com/tt/heroku-docker/blob/3e36cf75f0c822e937675cf0a4206248132b7c47/platforms/ruby/Dockerfile.t#L5).